test/OWNERS

```diff
@@ -2,3 +2,5 @@
 
 emeritus_approvers:
 - trasc
+reviewers:
+- kannon92
```

Comment:
```suggestion
reviewers:
- kannon92

emeritus_approvers:
- trasc
```

nits: It would be better to put active owner at first.

Reply:
done.